bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCreateTable.java

```diff
@@ -113,4 +117,58 @@ public void testTableNames() throws IOException {
       }
     }
   }
+  
+  @Test
+  public void testSplitKeys() throws IOException {
+    byte[][] splits = new byte[][] {
+        Bytes.toBytes("AAA"),
+        Bytes.toBytes("BBB"),
+        Bytes.toBytes("CCC"),
+    };
+    
+    Admin admin = getConnection().getAdmin();
+    
+    TableName tableName = TableName.valueOf("TestTableWithSplits");
+    HTableDescriptor descriptor = new HTableDescriptor(tableName);
+    descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY));
+    
+    try {
+      if (admin.tableExists(tableName)) {
+        LOG.warn("Not creating the table since it exists: %s", tableName);
+      } else {
+        admin.createTable(descriptor, splits);
+        RegionLocator locator = getConnection().getRegionLocator(tableName);
+        List<HRegionLocation> regions = locator.getAllRegionLocations();
+        // The number of regions should be the number of splits + 1.
+        Assert.assertEquals(splits.length + 1, regions.size());
+        for (int i = 0; i < regions.size(); i++) {
+          HRegionLocation region = regions.get(i);
+          String start_key = new String(region.getRegionInfo().getStartKey());
```

Comment:
HBase has a utility that's better for byte[] -> String Conversion.  Bytes.toString(region.getRegionInfo().getStartKey().  It's better to use than new Striong(bytes). (although I'm not sure why... probably encoding)